src/components/JourneySelector.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import '../App.css';
+import { fetchFares } from '../helpers/ApiCallHelper';
+import { DepartureInfo } from '../models/DepartureInfo';
+import DropdownList from './DropdownList';
+
+const JourneySelector: React.FC = () => {
+    const stations = [
+        { crs: 'KGX', name: 'King\'s Cross' }, 
+        { crs: 'SVG', name: 'Stevenage' }, 
+        { crs: 'YRK', name: 'York' }, 
+        { crs: 'DAR', name: 'Darlington' }, 
+        { crs: 'NCL', name: 'Newcastle' },
+    ];
+
+    const [departureValue, setDepartureValue] = useState(stations[0]);
+    const [arrivalValue, setArrivalValue] = useState(stations[0]);
+
+    const journeys: Array<DepartureInfo> = [];
```

Comment:
nitpick, purely a different conventions thing, both do the same thing but I think the below is more common,

using [] instead of array.
```suggestion
    const journeys: DepartureInfo[] = [];
```